lib/checkcondition.cpp

```diff
@@ -784,8 +784,9 @@ static inline T getvalue(const int test, const T value1, const T value2)
         return value2;
     case 5:
         return std::numeric_limits<T>::max();
+    default: 
```

Comment:
I don't see why a default is added in any of the switch bodies so please explain or remove them..